Mathlib/Data/Int/Bitwise.lean

```diff
@@ -94,7 +94,7 @@ theorem bodd_mul (m n : ℤ) : bodd (m * n) = (bodd m && bodd n) := by
   simp only [negSucc_coe, bodd_neg, bodd_coe, ←Nat.bodd_mul]
 -- Porting note: Heavily refactored proof, used to be:
 -- `by cases m with m m; cases n with n n;`
--- `simp [← int.mul_def, int.mul, -of_nat_eq_coe, bool.bxor_comm]`
+-- `simp [← Int.mul_def, int.mul, -of_nat_eq_coe, Bool.xor_comm]`
```

Comment:
The other two may be off too

Reply:
This is a quoted Lean 3 proof, so I'm going to revert this chunk.